### ISSUE TITLE
Error deploying Vision AI Application with custom trained model

### ISSUE THREAD

**Issue by pkim-inmotion** [google/visionai#13]
Hi,

I have been using Vision AI platform for a while with a custom trained model.
Our model has been updated and registered registered on 2023-02-09 and has been working correctly.

I could undeploy the application on 2023-05-15 to adjust the input streams of the application and could deploy without problem.
However, I found out that there was mismatching information in the metadata sent to our model. I went into the application view panel and found out that the entire application setup UI is changed. 

Now, there is "Universal input" instead of "Streams" and I could see that the Universal input node was not connected to my model. In order to fix that, I undeployed the application and deleted the application.

I followed the step by step guide published on https://cloud.google.com/vision-ai/docs/build-app. However, the "Add streams input" section does not match to the UI and I could not follow the steps.
Finally, I registered the streams as "Streaming" and registered 3 cameras to test. I added my custom trained model, with all the options, as an "Output" of the "Universal input" node and I could see that the two nodes were connected on the graph.

Finally I tried to Deploy the application and received a deployment error. I tried different combinations, including the deployment by API, but without a positive result. I am attaching the deployment error.
{
    "name": "projects/c5i-data-lake/locations/us-central1/operations/operation-1684582139755-5fc1e5808fc81-2f7c5bcc-5d50bdbc",
    "metadata": {
        "@type": "type.googleapis.com/google.cloud.visionai.v1.OperationMetadata",
        "createTime": "2023-05-20T11:28:59.981037429Z",
        "endTime": "2023-05-20T11:40:47.157448423Z",
        "target": "projects/c5i-data-lake/locations/us-central1/applications/c5i-20230520-placas",
        "verb": "update",
        "statusMessage": "Custom Model Deployment Step 2 Pending....",
        "requestedCancellation": false,
        "apiVersion": "v1"
    },
    "done": true,
    "error": {
        "code": 3,
        "message": "Instance Deployment Step 1 failed, step: projects/c5i-data-lake/locations/us-central1/applications/c5i-20230520-placas/instances/c5i-test-pedernales-lpr-DeployInstance. Detailed App Platform Error: Invalid application instance graph, it cannot be deployed:  Detailed App Platform Error:  Invalid Application Instance, there should only be 1 stream input, but now: 0"
    }
}
I am also attaching two images from the UI.
![Screenshot 2023-05-20 081045](https://github.com/google/visionai/assets/121249563/1a6d3bf5-f6db-479c-a578-c88faa52f28d)
![Screenshot 2023-05-20 080943](https://github.com/google/visionai/assets/121249563/ae05e67b-6e01-4c1d-a8e6-52f7ca99f072)

**Comment by dchao34** [comment]
Thanks for reporting; we are looking into it.

**Comment by pkim-inmotion** [comment]
Will the documentation be updated according to the final fix?

**Comment by dchao34** [comment]
Yeah, though it might trail the fix by a few days, because the documentation update is part of a launch train. It will be out some time next week.
